src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/synthetic_image_generator.py

```diff
@@ -1,62 +1,82 @@
-import base64
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import glob
+import random
 from enum import Enum, auto
-from io import BytesIO
-from typing import Optional, cast
+from pathlib import Path
 
-import numpy as np
+from genai_perf import utils
 from PIL import Image
 
 
 class ImageFormat(Enum):
-    JPEG = auto()
     PNG = auto()
+    JPEG = auto()
 
 
 class SyntheticImageGenerator:
-    def __init__(
```

Comment:
@mwawrzos I changed to using classmethod because I noticed that in LlmInputs, `create_synthetic_image` needs to be called multiple times along with `create_synthetic_prompt` so it made more sense to make it classmethod, rather than creating an instance of object (somewhere in the upper stack and passing the object all the way down to call its method).

Reply:
Ah, I see your point. It is a consequence of making the `‎LlmInputs.create_llm_inputs` a classmethod before.